Hardware/Hardware/CardReader/CardReaderService.swift

```diff
@@ -16,6 +16,9 @@ public protocol CardReaderService {
     /// The Publisher that emits software update state changes
     var softwareUpdateEvents: AnyPublisher<CardReaderSoftwareUpdateState, Never> { get }
 
+    /// The Publisher that emits when TTP Terms and Services are accepted
+    var builtInCardReaderAcceptToSEvents: AnyPublisher<Void, Never> { get }
+
```

Comment:
> Since this only happens during connection, and usually happens right before a software update, would it be convenient to include it as part of CardReaderSoftwareUpdateState, using associated values?
> I suppose that isn't great type design, because it won't be relevant to most software updates.

I agree. Software update events are not related to ToS acceptance.

> Alternatively perhaps, change the signature of func connect(_ reader: options:) -> AnyPublisher<CardReader, Error> to return a publisher with more detailed events informing about the connection process, including this one?

Thanks. I'll investigate how this `connect` method looks and works and where the ToS would fit. 
I only reviewed `connectedReaders`, `discoveredReaders`, `readerEvents`, and `softwareUpdateEvents` publishers and concluded that ToS event doesn't particularly fit in any. The closest that I think is `public var readerEvents: AnyPublisher<CardReaderEvent, Never>`.